frontend/src/metabase/admin/datamodel/containers/FieldApp.jsx

```diff
@@ -155,7 +149,6 @@ export default class FieldApp extends React.Component {
     };
   };
 
-  onUpdateField = this.linkWithSaveStatus(this.props.updateField);
```

Comment:
As far as I can tell, `onUpdateField` had the same effect as `onUpdateFieldProperties` but without some error handling. I merged them.